Source/Shaders/Builtin/Functions/cascadeDistance.glsl

```diff
@@ -1,4 +1,7 @@
+
+uniform vec4 u_shadowMapCascadeDistances;
```

Comment:
We normally don't use uniforms like this. The `czm_` uniforms are set once per frame except `czm_model*`. The other option was passing them as arguments. The downside was that there was a lot of parameters for each `czm_` function. Perhaps put them all in a `struct` and use that as a parameter?